minisat/core/Solver.cc

```diff
@@ -763,19 +763,19 @@ lbool Solver::search(int nof_conflicts)
 
             if (--learntsize_adjust_cnt == 0){
                 learntsize_adjust_confl *= learntsize_adjust_inc;
-                learntsize_adjust_cnt    = (int)learntsize_adjust_confl;
+                learntsize_adjust_cnt    = (size_t)learntsize_adjust_confl;
                 max_learnts             *= learntsize_inc;
 
                 if (verbosity >= 1)
-                    printf("c | %9d | %7d %8d %8d | %8d %8d %6.0f | %6.3f %% |\n",
-                           (int)conflicts, 
-                           (int)dec_vars - (trail_lim.size() == 0 ? trail.size() : trail_lim[0]), nClauses(), (int)clauses_literals, 
-                           (int)max_learnts, nLearnts(), (double)learnts_literals/nLearnts(), progressEstimate()*100);
+                    printf("| %9ld | %7ld %8ld %8ld | %8ld %8ld %6.0f | %6.3f %% |\n",
```

Comment:
This misses the "c " prefix. Furthermore, could you please adapt the %ld into another type, e.g. %zu?